maetio/next-web
# Added Basic stripe example into webstack

Testing instructions
1. signIn 
2. on home screen after signing in click 'stripe' button
3. wait a few seconds for the stripe UI to load
4. test out payment by typing 42424242... for every single card information box (this is a test card number provided by stripe)
5. click 'checkout' or whatever the button is called below stripe UI
6. should route you to a random screen if it worked.
